scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -312,8 +312,8 @@ def test_area_reconstitution(self, n, dim, radius, shift,
                                  single_hemisphere):
         # sample points uniformly on the hypersphere
         rng = np.random.RandomState(seed=0)
-        points = rng.normal(-1, 1, (n, dim))
-        points /= np.linalg.norm(points, axis=1).reshape((n, 1))
+        points = rng.randn(n, dim)
+        points /= np.linalg.norm(points, axis=1)[:, np.newaxis]
```

Comment:
you can get this behavior via `keepdims=True`

Reply:
Nice - this is a great feature.
I will save this for my next PR. I need to make this change in a few places and this is already quite a big PR to review.